### DISCUSSION TITLE
Respectfully requesting new "release" tag.

### DISCUSSION THREAD

**Discussion by jeffbowman** [magit/forge#596] [Support] [answered]
I don't follow development and prefer to use the "stable" versions from MELPA Stable.  The current version is broken related to fixes contained in #535.  Could we please have a new "release" which includes those fixes? 

Thank you very much!

N.B.  I started to put this in as an "issue" but as this is not a defect/bug, nor a feature request, it didn't seem appropriate there.  Hopefully this is the correct location.

**Comment by hab25** [comment]
This is not an answer, only a comment.

I second this request. 

Further, I'd like to ask: What is the strategy behind when to make a new stable release? Is this strategy being followed? If not, is that due to a lack of time?

I only ask these so that I, as a user, may, with this information, better choose between stable and unstable across all of @tarsius's packages.

Thank you for your great work!

<details> <summary>The following relates to another critical bug in stable and is probably not useful; I expect this to have been fixed in unstable by now</summary>

Reproduce it with `emacs -Q`, after first making sure `forge` is in `load-path` and at the current stable version (v0.3.2):
```
(progn
  (setopt debug-on-error t)
  (require 'forge)
  (forge-reset-database)
  (forge-list-repositories))
```

Backtrace:
```
Debugger entered--Lisp error: (wrong-number-of-arguments (1 . 3) 4)
  #f(compiled-function (class &optional livep connection-class) #<bytecode 0x1af8a0fa0f3016e0>)(forge-database forge--db-connection "/home/hab25/.config/emacs/forge-database.sqlite" t)
  apply(#f(compiled-function (class &optional livep connection-class) #<bytecode 0x1af8a0fa0f3016e0>) forge-database (forge--db-connection "/home/hab25/.config/emacs/forge-database.sqlite" t))
  closql-db(forge-database forge--db-connection "/home/hab25/.config/emacs/forge-database.sqlite" t)
  forge-db()
  forge-sql([:select $i1 :from repository :order-by [(asc owner) (asc name)]] [id owner name sparse-p selective-p worktree])
  forge-repository-list-refresh()
  forge-repository-list-setup(forge-repository-list-refresh "*Forge Repositories*")
  forge-list-repositories()
  (progn (setopt--set 'debug-on-error t) (require 'forge) (forge-reset-database) (forge-list-repositories))
  (progn (progn (setopt--set 'debug-on-error t) (require 'forge) (forge-reset-database) (forge-list-repositories)))
  elisp--eval-last-sexp(t)
  eval-last-sexp(t)
  eval-print-last-sexp(nil)
  funcall-interactively(eval-print-last-sexp nil)
  command-execute(eval-print-last-sexp)
```
</details>

**Comment by tarsius** [comment]
I'm releasing in May.

I'm generally not good at planning releases, but this time around it is particularly difficult to pull of because I have to release `magit`, `forge`, `emacsql` and all their dependencies and dependants at the same time. Everything is ready now, except forge. For the last nine months or so I have tried to only work on the things that move me closer to the goal of releasing. Unfortunately in the case of Forge, after overcoming major hurdles, I did fall into the trap of "just one more thing". I am confident this last thing, I am currently working on, really is the last.

Going forward I plan to release every month or every other month.

**Reply by tarsius** [reply, marked as answer]
The plan was to release on New Years Day, by the way. But I got sick for two weeks around that date and once the date had swooshed by, I had lost my focus and decided to finish a few in progress forge improvements first. This has been dragging on a bit too much.

**Reply by hab25** [reply]
Thank you for the answer!

Once you settle on a release strategy/cadence, it may be worthwhile to document it; it is user-facing in that users always have to choose (sometimes implicitly) which version to install.
